### PR TITLE
task2

### PR DESCRIPTION
**Pull request by myrrhfawn**

### CODE REVIEW

File: bin/arduino-cli
Comment (andrii-s-spitzer-lpnu-ua) [inline bin/arduino-cli:1]: remove bin files from git

**Comment by andrii-s-spitzer-lpnu-ua** [comment]
can't find actions

File: bin/arduino-cli
Comment (myrrhfawn) [reply bin/arduino-cli:1]: done

**Comment by myrrhfawn** [comment]
> can't find actions

done

**Comment by andrii-s-spitzer-lpnu-ua** [comment]
can't find artifacts

**Comment by myrrhfawn** [comment]
> can't find artifacts

fixed

**Review by andrii-s-spitzer-lpnu-ua — APPROVED** [review]

**Pull request merged**